pkgs/applications/science/logic/lean4/default.nix

```diff
@@ -31,6 +31,11 @@ stdenv.mkDerivation rec {
     patchShebangs stage0/src/bin/ src/bin/
   '';
 
+  postInstall = ''
+    mkdir -p $out/share/doc/lean4
+    mv $out/LICENSE* $out/share/doc/lean4
+  '';
+
```

Comment:
Would you mind using the following instead?
```diff
--- a/pkgs/applications/science/logic/lean4/default.nix
+++ b/pkgs/applications/science/logic/lean4/default.nix
@@ -46,6 +46,7 @@ stdenv.mkDerivation rec {
 
   cmakeFlags = [
     "-DUSE_GITHASH=OFF"
+    "-DINSTALL_LICENSE=OFF"
   ];
```

Reply:
Sure.